datachannel.go

```diff
@@ -73,11 +73,22 @@ func NewDataChannel(o unsafe.Pointer) *DataChannel {
 	return c
 }
 
-func (c *DataChannel) Send(data []byte) {
+// Send a message over a datachannel in either text or binary mode.
+func (c *DataChannel) Send(data []byte, binary bool) {
```

Comment:
I'd rather not change the signature of this method.  It'd be fine to add `SendText()` and have both call an unexported `sendInternal()` with a bool.